Plugins/Flow.Launcher.Plugin.BrowserBookmark/Flow.Launcher.Plugin.BrowserBookmark.csproj

```diff
@@ -64,14 +64,16 @@
     <Compile Remove="Bookmark.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Windows.Forms">
+      <HintPath>..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Windows.Forms.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```

Comment:
this should not be needed. If you want to use windows form, just add `<UseWindowsForms>true</UseWindowsForms>`.

Reply:
Fixed